packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -42,34 +44,77 @@ function genConfig(compatVersion: number): CompatConfig[] {
         }];
     }
 
-    return [
+    const allOld = {
+        loader: compatVersion,
+        driver: compatVersion,
+        containerRuntime: compatVersion,
+        dataRuntime: compatVersion,
+    };
+
+    const configs: CompatConfig[] = [
         {
-            name: `compat N${compatVersion} - old loader + new runtime`,
+            name: `compat N${compatVersion} - old loader`,
             kind: CompatKind.Loader,
             compatVersion,
             loader: compatVersion,
+            // back-compat: remove driver in future releases
+            driver: compatVersion,
+        },
+        {
+            name: `compat N${compatVersion} - new loader`,
+            kind: CompatKind.Loader,
```

Comment:
The compat kind was intented to be unique and provide a quick way for test to disable certain configurations, although I don't think it is being used at the moment, so probably ok for now.

Long term, we might want to have a better way to declare the min version that a test should be enable for.